src/components/SearchForm.jsx

```diff
@@ -68,42 +63,40 @@ const SearchForm = ({
       type="button"
       onClick={() => console.log('clicked clear')}
     >Clear</button>
-  <button
-    type="button"
-    onClick={() => console.log('clicked "Save Search"')}
-  >Save Search</button>
   </Form>
 )
 
-
-export const FormikSearch = withFormik({
+const FormikSearch = withFormik({
   mapPropsToValues: ({
     searchTerms,
     location,
     radius,
     timeFrame,
     savedSearch,
-    plan,
   }) => {
     {/* return an object which sets initial values of the form */}
     return {
       searchTerms: searchTerms || '',
       location: location || '97015',
       radius: radius || '',
       timeFrame: timeFrame || '',
-      savedSearch: savedSearch || true,
-      plan: plan || 'free',
+      savedSearch: savedSearch || false,
     }
   },
   validationSchema: Yup.object().shape({
     searchTerms: Yup.string().required('must include at least one search term'),
     location: Yup.string().matches(/^[0-9]{5}$/, 'Location must be a valid 5 digit zipcode'),
-    radius: Yup.string(),
+    radius: Yup.number().positive('Radius must be a number'),
     timeFrame: Yup.string(),
   }),
-  handleSubmit(values) {
-    {/* this is where we would run dispatch or make an AJAX call */}
-    console.log(values)
+  handleSubmit(values, FormikBag) {
+    FormikBag.props.submit(values)
   }
```

Comment:
Turns out Formik has this FormikBag object, which according to the documentation "contains a subset of the injected props and methods as well as any props that were passed to the wrapped component" (https://jaredpalmer.com/formik/docs/api/withformik#handlesubmit-values-values-formikbag-formikbag-void)

Formik needs to access that in order to handle form submission.